libkbfs/bare_root_metadata.go

```diff
@@ -516,10 +516,37 @@ func (md *BareRootMetadata) getTLFKeyBundles(keyGen KeyGen) (
 	return &md.WKeys[i], &md.RKeys[i], nil
 }
 
+// GetDeviceKIDs returns the KIDs (of CryptPublicKeys) for all known
+// devices for the given user at the given key generation, if any.
+// Returns an error if the TLF is public, or if the given key
+// generation is invalid.
+func (md *BareRootMetadata) GetDeviceKIDs(
```

Comment:
Hmm, I don't think so. Users of `KeyMetadata` just need `HasKeyForUser` (which was intended to replace this), and server code already works with RootMetadataSigned objects directly.